governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,241 @@
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084)
+
+If this executive proposal passes, the following address will be whitelisted in the `RWA015-A` output conduit:
+
+- [0x6759610547a36E9597Ef452aa0B9cace91291a2f](https://etherscan.io/address/0x6759610547a36E9597Ef452aa0B9cace91291a2f)
+
+### Push USDP out of the input conduit
+
+- **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084/3)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084)
+
+If this executive proposal passes, following the recommendation of the Governance Facilitators in collaboration with the spell team, the following operations will be performed:
+
+- Raise the PSM_PAX_A Debt Ceiling to **100,000 DAI**.
+- Call `push(uint256 amt)` on the the PSM_PAX_A_INPUT_CONDUIT_JAR contract with an amount of **84,211.27 USDP**, depositing the contents of the contract to the PSM and receiving an identical amount of Dai into the PSM_PAX_A_JAR contract.
+- Call `void()` on the PSM_PAX_A_JAR contract to deposit the Dai to the Surplus Buffer.
+- Set PSM-PAX-A Debt Ceiling to **0 DAI**.
+- Decrease Debt Ceiling of PSM_PAX_A back to **0 DAI**.
+
+This will have the following net results:
+
+- The PSM_PAX_A Debt Debt Ceiling remains at 0 DAI after the transaction has been executed.
+- Dai generated from PSM_PAX_A increases by **84,211.27 DAI**.
+- USDP balance of MCD_JOIN_PSM_PAX_A increases by **84,211.27 USDP**.
+- USDP balance of PSM_PAX_A_INPUT_CONDUIT_JAR decreases to **0 USDP**.
+- Surplus Buffer increases by **84,211.27 DAI**.
+
+### Trigger Spark Proxy Spell
+
+- **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033/7), Governance Polls [1095](https://vote.makerdao.com/polling/QmfGV2vt), [1096](https://vote.makerdao.com/polling/QmSYZSCQ), [1097](https://vote.makerdao.com/polling/QmcRdMyA), [1098](https://vote.makerdao.com/polling/QmSh8gyC), [1099](https://vote.makerdao.com/polling/QmUhT32b), [1100](https://vote.makerdao.com/polling/QmZND8WW), [1104](https://vote.makerdao.com/polling/QmVXriiT), [1106](https://vote.makerdao.com/polling/QmVsKsGa)
+- **Proposal:** [Forum post "[Apr 4, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033), [Forum post "SparkLend External Security Access Multisig for Freezer Mom"](https://forum.makerdao.com/t/sparklend-external-security-access-multisig-for-freezer-mom/24070)
+
+If this executive proposal passes, the SparkLend Proxy Spell on Ethereum and Gnosis at [0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882](https://etherscan.io/address/0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882) will be executed, containing the following items:
```

Comment:
:green_circle: **nit, non-blocking**

1. Technically the Proxy Spell is not *on* Gnosis Chain but is deployed and executed on mainnet with the changes passed through a governance relay.
2. Very minor, but we should agree a consistent styling for the Proxy Spells. Is it a Spark Protocol Proxy Spell? A Spark Proxy Spell? A SparkLend Proxy Spell? et cetera